CardForm/src/main/java/com/braintreepayments/cardform/utils/CardType.java

```diff
@@ -35,7 +35,7 @@
             R.drawable.bt_ic_jcb,
             16, 16,
             3, R.string.bt_cvv, null),
-    MAESTRO("^(5018|5020|5038|5[6-9]|6020|6304|6703|6759|676[1-3])\\d*",
+    MAESTRO("^(5018|5020|5038|5043|5[6-9]|6020|6304|6703|6759|676[1-3])\\d*",
```

Comment:
This looks like it's missing `49358` https://github.com/braintree/credit-card-type/blob/master/src/lib/card-types.ts#L111

And some other ranges (though I don't know how easy it is for ya'll to handle ranges in this lib as is) https://github.com/braintree/credit-card-type/blob/master/src/lib/card-types.ts#L112-L114

Reply:
Yeah adding `49358` shouldn't be hard, but we don't have range support on any of the cards on Android. I wonder if it's something to consider in another PR where we try to reach parity with the pan gem.

Reply:
This PR will help address a GH issue in drop-in where the leading `5043` Maestro number was identified as not having support.

Reply:
We added `49358` to the enum list but had to re-order how they're validated because VISA_REGEX `^4` was detecting the Maestro number as Visa.

Reply:
I'm a little confused -- are we not going to add `49358` in this PR? I don't see it in the diff anymore.